config/initializers/inflections.rb

```diff
@@ -0,0 +1,3 @@
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym "URL"
```

Comment:
It has to be the whole class name. See how we did `URIService`.

Reply:
I'm a bit hesitant to add that acronym thinking that it's going to affect all classes in the parent app.
Maybe the developer does not want to sue that inflection rule on their code.

Reply:
It is probably because "[Generators use the Active Support Inflector](https://github.com/rails/rails/issues/44075#issuecomment-1005914997)". So I don't see any other solution then renaming the helper not to use any acronyms (and actually trying to avoid this throughout the whole gem).